meltano.yml

```diff
@@ -18,16 +18,19 @@ plugins:
         file: alphabet.csv
         keys:
```

Comment:
```suggestion
        path: ./tap_csv/tests/data/
        keys:
```

Reply:
@TyShkan I noticed that path was missing in the meltano.yml and was throwing an error when I tried running it. Figured we should add it while doing all this cleanup already.